webpack-and-babel/imports.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-unused-vars */
+
+// import from a file located at the absolute path `/foo/bar/baz/qux.js`:
+// eslint-disable-next-line import/no-absolute-path
+import f1 from '/foo/bar/baz/qux.js';
+
+// import from a file named `foo.json` located in the current directory:
+import f2 from 'foo.json';
+
+// import from a file named `bar.js` located in the parent directory:
+import f3 from '../bar.js';
+
+// import from a file named `index.js` located in a directory named `baz` that
+// is in the current directory:
+import f4 from './baz/index.js';
+
+// import from a file named `index.js` located in a directory named `baz` that
+// is in the parent directory:
+import f5 from '../baz/index.js';
```

Comment:
You do not need to include `/index`. If webpack is not able to find an export automatically by adding the file extensions, it will then look for an `index.js` file automatically. So we don't even need to include it